static/ankersrv.js

```diff
@@ -355,18 +426,31 @@ $(function () {
 
     sockets.ctrl = new AutoWebSocket({
         name: "Control socket",
-        url: `ws://${location.host}/ws/ctrl`,
+        url: `${location.protocol.replace('http','ws')}${location.host}/ws/ctrl`,
         badge: "#badge-ctrl",
     });
 
+    sockets.pppp_state = new AutoWebSocket({
+        name: "PPPP socket",
+        url: `ws://${location.host}/ws/pppp-state`,
```

Comment:
Found an error in the pppp_state that would prevent PPPP connection over https:

```suggestion
        url: `${location.protocol.replace('http','ws')}${location.host}/ws/pppp-state`,
```

Reply:
@treitmayr This was from your changes in this PR:
https://github.com/Ankermgmt/ankermake-m5-protocol/pull/145/commits/1089b493d13e6c3b6abbe4a9168972407702e8c4

Reply:
My changes were done in a consistent way in the existing code base at the time of the commit. However, the merge of [d6c727b](https://github.com/Ankermgmt/ankermake-m5-protocol/pull/160/commits/d6c727b378140d58395d50f12482fec53864b1d3) introduced wss support, again on the code base from back then. This is the explanation for the current state.
However, I agree with @sondregronas to change the affected line in the proposed way.